mmv1/third_party/terraform/utils/bootstrap_utils_test.go

```diff
@@ -418,12 +418,17 @@ func removeContainerServiceAgentRoleFromContainerEngineRobot(t *testing.T, proje
 	}
 }
 
-func BootstrapProject(t *testing.T, projectID, billingAccount string, services []string) *cloudresourcemanager.Project {
+// BootstrapProject will create or get a project named "<projectIDPrefix><testProjectFromEnv>"
```

Comment:
Is a prefix better than a suffix for distinguishing bootstrapped projects?

Reply:
I honestly couldn't come up with a good reason for one over the other. The prefix, as I have it written, will make the projects a little more clearly "tf-" test resources (ie. they can be deleted/recreated if necessary), but a suffix might be slightly nicer for grouping alphabetically. That said, I would still lean toward the prefix strategy to start.